packages/components/Menu/src/MenuList/MenuList.tsx

```diff
@@ -51,20 +52,24 @@ export const MenuList = compose<MenuListType>({
       const content =
         Platform.OS === 'macos' ? (
           <Slots.root>
-            <Slots.focusZone
-              componentRef={focusZoneRef}
-              focusZoneDirection={'vertical'}
-              defaultTabbableElement={focusZoneRef}
-              // eslint-disable-next-line @typescript-eslint/ban-ts-comment
-              // @ts-ignore FocusZone takes ViewProps, but that isn't defined on it's type.
-              enableFocusRing={false}
-              forceFocusMacOS={true}
-            >
-              {children}
-            </Slots.focusZone>
+            <Slots.scrollView>
+              <Slots.focusZone
+                componentRef={focusZoneRef}
+                focusZoneDirection={'vertical'}
+                defaultTabbableElement={focusZoneRef}
+                // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+                // @ts-ignore FocusZone takes ViewProps, but that isn't defined on it's type.
+                enableFocusRing={false}
+                forceFocusMacOS={true}
```

Comment:
I think this no longer exists so we can get rid of it? 😅

Reply:
Removed